### PR TITLE
[ONNX] Support constant 'aten::__contains__'

### LINKED ISSUE #83661: [ONNX] Support exporting new dispatch to linalg.vector_norm and linalg.matrix_norm from torch.norm
### 🐛 Describe the bug

As per title.

### Versions

https://github.com/pytorch/pytorch/pull/81761

### PR DESCRIPTION
**Pull request by BowenBao**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #91660

#84624 introduces an update on `torch.norm` [dispatch logic](https://github.com/pytorch/pytorch/blob/eaa43d9f254fa01ee2d61dc2932c4e48987d7f96/torch/functional.py#L1489) which now depends on `layout`. Resulting in regressions to export related operators from TorchScript. 

This PR resolves the regression by partially supporting a subset use case of `prim::layout` (only `torch.strided`), `aten::__contains__` (only constants) operators. It requires much more effort to properly support other layouts, e.g. `torch.sparse_coo`. Extending JIT types, and supporting related family of ops like `aten::to_sparse`. This is out of the scope of this PR.

Fixes #83661

### CODE REVIEW

**Comment by justinchuby** [comment]
Some tests failing

**Comment by BowenBao** [comment]
> Some tests failing

Messed up lint fix, should be fine now.

File: torch/onnx/symbolic_opset9.py [hunk 1]
```diff
@@ -6018,6 +6018,27 @@ def dim(g: jit_utils.GraphContext, self):
...
+def __contains_(g: jit_utils.GraphContext, self, element):
+    unpacked_list = symbolic_helper._unpack_list(self)
+    if all(
+        [symbolic_helper._is_constant(x) for x in unpacked_list]
+    ) and symbolic_helper._is_constant(element):
+        return g.op(
+            "Constant",
+            value_t=torch.tensor(
+                symbolic_helper._node_get(element.node(), "value")
+                in (symbolic_helper._node_get(x.node(), "value") for x in unpacked_list)
```
Comment (justinchuby) [inline torch/onnx/symbolic_opset9.py:6032]: This is a little complex. Can it be written out as a for loop?

**Review by justinchuby — APPROVED** [review]

**Comment by BowenBao** [comment]
@kit1980 please help approve. The changes outside of onnx export is extending a torch script pybind method `ListType::ofStrings`

**Review by kit1980 — APPROVED** [review]

**Pull request closed without merging**